### PR TITLE
Update docker stack install instructions

### PR DESCRIPTION
**Pull request by johnsonw**
Signed-off-by: Will Johnson <wjohnson@whamcloud.com>

### CODE REVIEW

File: docs/Install_Guide/ig_docker_stack.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ Starting with Integrated Manager for Lustre 5, the IML manager is available via
     # This must be done due to: https://success.docker.com/article/ipvs-connection-timeout-issue
     echo net.ipv4.tcp_keepalive_time = 600 >> /etc/sysctl.conf
     sysctl -p
+    wget http://mirror.centos.org/centos/7/extras/x86_64/Packages/container-selinux-2.107-1.el7_6.noarch.rpm
```
Comment (jgrund) [inline docs/Install_Guide/ig_docker_stack.md]: Why not just add the extras repo? This link will go stale fairly quickly.

Comment (johnsonw) [reply docs/Install_Guide/ig_docker_stack.md]: sure

Comment (ip1981) [reply docs/Install_Guide/ig_docker_stack.md]: I also would like to notice http and absence of any integrity check.

File: docs/Install_Guide/ig_docker_stack.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ Starting with Integrated Manager for Lustre 5, the IML manager is available via
     # This must be done due to: https://success.docker.com/article/ipvs-connection-timeout-issue
     echo net.ipv4.tcp_keepalive_time = 600 >> /etc/sysctl.conf
     sysctl -p
```
Comment (jgrund) [inline docs/Install_Guide/ig_docker_stack.md]: This is no longer needed.

File: docs/Install_Guide/ig_docker_stack.md [hunk 3]
```diff
@@ -12,6 +12,8 @@ Starting with Integrated Manager for Lustre 5, the IML manager is available via
...
+    rpm --import http://mirror.centos.org/centos/RPM-GPG-KEY-CentOS-7
+    yum-config-manager --add-repo http://mirror.centos.org/centos/7/extras/x86_64/
```
Comment (jgrund) [inline docs/Install_Guide/ig_docker_stack.md]: I question if we need to add this. Extras should be available on a stock install.

File: docs/Install_Guide/ig_docker_stack.md [hunk 4]
```diff
@@ -25,7 +27,7 @@ Starting with Integrated Manager for Lustre 5, the IML manager is available via
     wget https://raw.githubusercontent.com/whamcloud/integrated-manager-for-lustre/master/docker/docker-compose.yml
```
Comment (jgrund) [inline docs/Install_Guide/ig_docker_stack.md]: We should pick somewhere other then /tmp for this file.

File: docs/Install_Guide/ig_docker_stack.md [hunk 5]
```diff
@@ -35,6 +37,8 @@ Starting with Integrated Manager for Lustre 5, the IML manager is available via
          - "<STORAGE_SERVER_X_NAME>:<STORAGE_SERVER_X_IP>"
          - "<STORAGE_SERVER_Y_NAME>:<STORAGE_SERVER_Y_IP>"
          - ...
+       environment:
+         - "NTP_SERVER_HOSTNAME=10.73.10.1"
```
Comment (jgrund) [inline docs/Install_Guide/ig_docker_stack.md]:
```suggestion
         - "NTP_SERVER_HOSTNAME=<NTP_TIMESERVER_ADDRESS>"
```

File: docs/Install_Guide/ig_docker_stack.md (hunk 3 above)
Comment (johnsonw) [reply docs/Install_Guide/ig_docker_stack.md]: Ok, i'll remove it then.

File: docs/Install_Guide/ig_docker_stack.md (hunk 2 above)
Comment (johnsonw) [reply docs/Install_Guide/ig_docker_stack.md]: Remove:
```
echo net.ipv4.tcp_keepalive_time = 600 >> /etc/sysctl.conf
sysctl -p
```

File: docs/Install_Guide/ig_docker_stack.md (hunk 4 above)
Comment (johnsonw) [reply docs/Install_Guide/ig_docker_stack.md]: How about `~`?

Comment (johnsonw) [reply docs/Install_Guide/ig_docker_stack.md]: The other option is we don't change a directory at all and just let them put it where they want it.

Comment (jgrund) [reply docs/Install_Guide/ig_docker_stack.md]: `/etc/iml-docker/` ?

Comment (johnsonw) [reply docs/Install_Guide/ig_docker_stack.md]: :+1:

File: docs/Install_Guide/ig_docker_stack.md [hunk 6]
```diff
@@ -21,11 +18,12 @@ Starting with Integrated Manager for Lustre 5, the IML manager is available via
...
 
    ```sh
-    cd /tmp
+    mkdir -p /etc/iml-docker/
+    cd /etc/iml-docker/
     wget https://raw.githubusercontent.com/whamcloud/integrated-manager-for-lustre/master/docker/docker-compose.yml
    ```
 
-1. Create a new file `docker-compose.extra-hosts.yml`, adjacent to docker-compose.yml. It should contain host to ip mappings for each storage server, so the job-scheduler can map them correctly:
+1. Create a new file `docker-compose.overrides.yml`, adjacent to docker-compose.yml. It should contain host to ip mappings for each storage server and any other neceesary overrides, so the job-scheduler can map them correctly:
```
Comment (jgrund) [inline docs/Install_Guide/ig_docker_stack.md]:
```suggestion
1. Create a new file `docker-compose.overrides.yml`, adjacent to docker-compose.yml. It should contain host to ip mappings for each storage server and any other necessary overrides, so the job-scheduler can map them correctly:
```

**Pull request merged**